website/content/sdk/guides/intros/stocks/government.md

```diff
@@ -0,0 +1,167 @@
+|    |            |          |                                 |          |  Telecommunications                    |  Issues pertaining to telecommunications tower real estate - no specific legislation. Issues related to global telecommunications policy - no specific legislation.                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+|    |            |          |                                 |          |  Taxation/Internal Revenue Code        |  Issues related to international and domestic tax provisions as applied to real estate investment trusts and Public Law 115-97 (Tax Cuts and Jobs Act). |
+| 495 | 2022-10-20 | DKNG     | DRAFTKINGS INC.                                   |    20000 | Taxation/Internal Revenue Code                | Tax characterization of daily fantasy sports.                               |
+| 496 | 2022-10-20 | ELVT     | ELEVATE CREDIT SERVICE, LLC                       |    90000 | Banking                                       | Monitored issues concerning online lending proposals.                       |
+| 497 | 2022-10-20 | HGV      | HILTON GRAND VACATIONS                            |    20000 | Financial Institutions/Investments/Securities | GENERAL DISCUSSIONS RELATED TO DEBT SERVICING                               |
+| 498 | 2022-10-20 | SO       | SOUTHERN CALIFORNIA TRIBAL CHAIRMEN'S ASSOCIATION |    40000 | Indian/Native American Affairs                | Fee to trust issues. Issues surrounding federal programs for Indian tribes. |
+| 499 | 2022-10-20 | ONCS     | ONCOSEC MEDICAL INCORPORATED                      |        0 |                                         |                                                                             |
+
+### Contracts
+
+Get a list of contracts awarded to a company and the agency authorizing it.
+
+```python
+gov.contracts('AMT')
+```
+
+|    | Date       | Description                                                                                                                                                                                     | Agency                                |   Amount |
```

Comment:
Please reduce the size of this table, e.g. with `limit=5`. It is currently too large.

Reply:
done